soda/core/soda/execution/check/anomaly_metric_check.py

```diff
@@ -57,7 +57,6 @@ def __init__(
                 check_identity=self.create_identity(), limit=3
             )
             self.diagnostics = {}
-            self.cloud_check_type = "anomalyDetection"
```

Comment:
Anomaly detection check has different properties than a generic check, and just changing the type doesn't actually allow properties like `anomalyErrorSeverity` to get into our Backend.

Reply:
removing anomaly checks from this PR, this will be dealt with separately